src/app/task/task.ts

```diff
@@ -0,0 +1,13 @@
+import { Status } from './status';
+
+export class Task {
+    status: Status;
+
+    constructor(public name: string,
+                public description: string) {
+                this.status = Status.IN_PROGRESS;
+        }
+                public id?: number;
+}
```

Comment:
move this above constructor and remove questionmark since we want all tasks to have id